scrapers/GotFilled.yml

```diff
@@ -0,0 +1,73 @@
+name: GotFilled
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - gotfilled.com/tour/videos/
+    scraper: sceneScraper
+
+performerByURL:
+  - action: scrapeXPath
+    url:
+      - gotfilled.com/tour/models/
+    scraper: performerScraper
+
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //*[@id="info"]/h2/text()
+      Date:
+        selector: //*[@id="info"]/div[4]/div[3]/h5/text()
+        postProcess:
+          - parseDate: Jan 2, 2006
+      Details: //*[@id="info"]/div[1]/text()
+      Performers:
+        Name: //*[@id="info"]/div[2]/h3/a/text()
+      Tags:
+        Name: //*[@id="info"]/div[5]/div[2]/p/a/text()
+      Image:
+      # Images are a div with an inline style containing a background-image with
+      # a url
+      # Inline style exemple: `background-image: url("{IMAGE_URL}");`
+        selector: //*[@id="vjs_video_3"]/div[3]/@style
+        postProcess:
+          - replace:
+            - regex: .*url\("|"\);
+              with: ""
+      Studio:
+        Name:
+          fixed: GotFilled
+
+  performerScraper:
+    common:
+      $details: //*[@id="root"]/main/div/div[2]/div[2]
+    performer:
+      Name: $details/h2/text()
+      Birthdate: $details/div/ul/li[1]/p/text()
+      HairColor: $details/div/ul/li[6]/p/text()
+      EyeColor: $details/div/ul/li[5]/p/text()
+      Height:
+        selector: $details/div/ul/li[8]/p/text()
+        postProcess:
+          - feetToCm: true
+      Weight:
+        selector: $details/div/ul/li[7]/p/text()
+        postProcess:
+          - replace:
+            - regex: " lbs"
+              with: ""
+          - lbToKg: true
+      Measurements: $details/div/ul/li[4]/p/text()
+      Details: $details/p
+      Image: //*[@id="root"]/main/div/div[2]/div[1]/img/@src
```

Comment:
Can be refactored to `//*[@id="root"]/main/div/div[contains(@class, "model-info")]/div[contains(@class, "thumbnail")]/img/@src`

Reply:
yeah safer/better